pkgs/applications/editors/apostrophe/default.nix

```diff
@@ -10,14 +10,13 @@ let
   texliveDist = texlive.combined.scheme-medium;
 
 in stdenv.mkDerivation rec {
-  pname = "uberwriter";
-  version = "unstable-2020-01-24";
-
-  src = fetchFromGitHub {
-    owner  = pname;
-    repo   = pname;
-    rev    = "0647b413407eb8789a25c353602c4ac979dc342a";
-    sha256 = "19z52fpbf0p7dzx7q0r5pk3nn0c8z69g1hv6db0cqp61cqv5z95q";
+  pname = "apostrophe";
+  version = "unstable-2020-03-20";
+
+  src = fetchgit {
```

Comment:
Use `fetchFromGitlab`, it has an parameter for sites != gitlab.com as well.

Reply:
Missed that, thanks!